internal/key/soft_key.go

```diff
@@ -260,10 +260,9 @@ func (m *SoftKey) Encode() string {
 	return m.privKeyEncoded
 }
 
-// Saves the private key to disk with hex encoding.
+// Saves the already CB58-encoded private key to disk.
 func (m *SoftKey) Save(p string) error {
-	k := hex.EncodeToString(m.privKeyRaw)
-	return ioutil.WriteFile(p, []byte(k), fsModeWrite)
+	return ioutil.WriteFile(p, []byte(m.privKeyEncoded), fsModeWrite)
```

Comment:
We should really improve the naming within this file and add some comments or otherwise switch to using an external library instead of implementing this within this repo.

Reply:
This is a separate piece of work though.

Reply:
What do you mean exactly?

Reply:
Oh I see, yeah I agree, pks in plaintext isn't state-of-the-art